src/utils/rss/generate-feed.js

```diff
@@ -55,8 +58,11 @@ const generateItem = function generateItem(post) {
 const generateRSSFeed = function generateRSSFeed(siteConfig) {
   return {
     serialize: ({ query: { allGhostPost } }) =>
-      allGhostPost.edges.map((edge) =>
+      allGhostPost.edges.map((edge) =>{
+        console.log(edge);
         Object.assign({}, generateItem(edge.node))
```

Comment:
Pass `siteConfig` to `generateItem` function as second parameter to use it later.

Reply:
remove console.log